Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -221,6 +209,51 @@ TYPO3 log, the deprecations log and logs generated by extensions.
 :file:`vendor/`
 ---------------
 
+In this directory, which lies outside of
+the webroot, third-party dependencies that are not TYPO3 extensions are
+installed.
+
+If optional :ref:`Composer installers v4 <composer-installer-v4>`
+are used, TYPO3 extensions are also installed here.
+
+.. _composer-installer-v4:
+
+TYPO3 v11 with Composer installers v4
+=====================================
+
+The Composer plugin `typo3/cms-composer-installers` in version 4+ was created
```

Comment:
That is not really true, v4 was there before TYPO3 v12 with v5. In fact, v5 requires just PHP 8.1 as v4 can also be used with PHP 7.4 - no other changes. And v4-rc1 was released months before v5.

Reply:
I copied the sentence from above and Have no ideas about this as I was not following it. Could you try to improve this section?